views/confirm-company.html

```diff
@@ -52,6 +55,7 @@ <h1 class="govuk-heading-l">{{ i18n.confirmCompanyTitle }} </h1>
             },
             {
               key: {
+                id: "confirm-incorporation-date",
```

Comment:
Feels like these should have a consistent prefix, Ie always confirm-company rather than just confirm.

Reply:
I wasn't sure if 'confirm-company' should be the prefix as there are fields that start with 'company', but I do agree. Will make the change